src/colmap/sensor/models.h

```diff
@@ -372,6 +374,28 @@ struct ThinPrismFisheyeCameraModel
       CameraModelId::kThinPrismFisheye, "THIN_PRISM_FISHEYE", 2, 2, 8)
 };
 
+// Aria Glasses Camera Model
+//
+// Camera model with radial and tangential distortion coefficients and
+// additional coefficients accounting for thin-prism distortion.
+//
+// See
+// https://facebookresearch.github.io/projectaria_tools/docs/tech_insights/camera_intrinsic_models#the-fisheyeradtanthinprism-fisheye624-model
+//
+// Parameter list is expected in the following order:
+//
+//    fx, fy, cx, cy, k1, k2, k3, k4, k5, k6, p1, p2, sx1, sx2, sy1, sy2
+//
+// Note:
+//    Tangential coefficients p1, p2 correspond to p1, p0 in the Fisheye624 Aria
+// model. Thin prism coefficients sx1, sx2, sy1, sy2 correspond to s0, s1, s2,
+// s3 in the Fisheye624 Aria model.
+//
+struct AriaFisheyeCameraModel : public BaseCameraModel<AriaFisheyeCameraModel> {
```

Comment:
I suggest calling this model "FisheyeRadTanThinPrismModel". This is also how the Aria project calls it.